apps/mull-ui/src/app/pages/messages/event-messages/event-message-list.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../../components';
+import EventBullet from '../../../components/event-bullet/event-bullet';
+import './event-message-list.scss';
+
+/* eslint-disable-next-line */
+export interface EventMessageListProps {}
+
+/*TODO: Remove this and replace with query of event message channels*/
+
+const messageChannels = [
+  {
+    eventTitle: 'Clean up of Rogers Park',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 1, 8, 20, 30, 1),
+    id: 1,
+  },
+  {
+    eventTitle: 'Lets do this guys!',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 5, 8, 20, 30, 1),
+    id: 2,
+  },
+  {
+    eventTitle: 'My god please clean this...',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 7, 8, 20, 30, 0),
+    id: 3,
+  },
+];
+
+export const EventMessageList = (props: EventMessageListProps) => {
+  const [searchString, setSearchString] = useState<string>('');
+
+  return (
+    <div className="page-container">
+      <ChatHeader eventTitle="Event Messages" style={{ right: '0.625rem', marginTop: '1rem' }} />
```

Comment:
Cindy had a great suggestion for the header in Leo's PR. https://github.com/RGPosadas/Mull/pull/243/files#r584393398. This should fix the gap between the top nav and the header
![Image from iOS](https://user-images.githubusercontent.com/44760251/109523997-f8e21980-7a7d-11eb-8b60-5b930d2305d6.png)

Reply:
Ya this is a good suggestion, im gonna do this